packages/terra-file-path/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "terra-file-path",
+  "main": "lib/FilePath.js",
```

Comment:
Will this be the only component exported or do we know at this time if there will be additional exports?

Reply:
Just this I believe!